CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# v0.31.1 (_2019-06-10_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v0.31.0...v0.31.1)
+
+## FxA Client
+
+### What's new
+
+- Added a new method to help recover from invalid access tokens.
+  ([#1244](https://github.com/mozilla/application-services/pull/1244)) If the
+  application receives an an authentication exception while using a token
+  obtained through `FirefoxAccount.getAccessToken`, it should:
+  - Call `FirefoxAccount.clearAccessTokenCache` to remove the invalid token from the internal cache.
+  - Retry the operation after obtaining fresh access token via `FirefoxAccount.getAccessToken`.
+  - If the retry also fails with an authentication exception, then the user will need to reconnect
+    their account via a fresh OAuth flow.
+- `FirefoxAccount.getProfile` now performs the above retry logic automagically.
+  An authentication error while calling `getProfile` indicates that the user
+  needs to reconnect their account.
+  ([#1244](https://github.com/mozilla/application-services/pull/1244)
```

Comment:
I guess the "keys" thing is a purely internal change, but I wonder if we should call it out in the changelog for completeness, especially because it's one of the motivators for taking these changes.

Reply:
I'll note it.